pkgs/top-level/all-packages.nix

```diff
@@ -4796,7 +4796,9 @@ with pkgs;
 
   wayland-utils = callPackage ../tools/wayland/wayland-utils { };
 
-  wayland-proxy-virtwl = callPackage ../tools/wayland/wayland-proxy-virtwl { };
+  wayland-proxy-virtwl = callPackage ../tools/wayland/wayland-proxy-virtwl {
+    ocamlPackages = ocaml-ng.ocamlPackages_5_0;
```

Comment:
Why not OCaml 5.1, which is the latest available in `nixpkgs`?

Reply:
Because it doesn't work:

```
       error: ocaml-migrate-parsetree is not available for OCaml 5.1.0
```

Reply:
Sounds like a strong argument!